cloud/transport/client.go

```diff
@@ -22,6 +22,31 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const serviceConfig = `
+{
+	"methodConfig": [
+		{
+			"name": [
+				{
+					"service": "weaviate.cloud.internal.cluster.ClusterService", "method": "Apply"
```

Comment:
Could we add in the proto file a comment referencing this file+line and indication to update it if we change the service/method name ?

Reply:
Or instead of having a const string, make it that the service name is initialized with the service name from the gRPC struct in go. This way it's less static.

Reply:
i agree, we could also have the config in separate  json file 🤔

Reply:
Good Point!
I don't think we need to mention this proto file. The service config is defined where it is used, so it should be straightforward to change the config. The unit test will catch the failure when the method name changes. I think this is the simplest way, but we can further improve it if required